fees/ether-fi/index.ts

```diff
@@ -17,6 +22,72 @@ const getTotalSupply = async (options, target) => {
   });
 };
 
+const getStethFees = async (options, totalSteth) => {
+  const stethRebaseLogs = await options.getLogs({
+    target: STETH,
+    fromBlock: await options.getStartBlock(),
+    toBlock: await options.getEndBlock(),
+    eventAbi: "event TokenRebased(uint256 indexed reportTimestamp,uint256 timeElapsed,uint256 preTotalShares,uint256 preTotalEther,uint256 postTotalShares,uint256 postTotalEther,uint256 sharesMintedAsFees)",
+  });
+  for (const log of stethRebaseLogs) {
```

Comment:
why not take just the first log instead of looping?

Reply:
Nice that would be cleaner!

Reply:
can you make that change because the loop is misleading

Reply:
done